packages/glimmer-runtime/lib/dom/change-lists.ts

```diff
@@ -126,6 +134,27 @@ export const InputValuePropertyChangeList = {
   }
 };
 
+export const TextAreaValueChangeList = {
+  setAttribute(dom: DOMHelper, element: Element, attr: string, value: Opaque) {
+    let textArea = <HTMLTextAreaElement>element;
+    let normalizedValue = normalizeTextValue(value);
+
+    if(textArea.childNodes.length === 0) {
+      let textNode = dom.createTextNode(normalizedValue);
+      textArea.appendChild(textNode);
+    } else {
+      let currentValue = textArea.value;
```

Comment:
I think it could. How would you expect this to render when `someValue` is `hello`?

```hbs
<textarea value={{someValue}}>why would someone put text here too?</textarea>
```